com.todesk.ToDesk.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: com.todesk.ToDesk
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: todesk.sh
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --own-name=com.todesk.toc.ui
```

Comment:
Why don't you submit using this ID? If that's what they use internally.

Reply:
I do not think com.todesk.toc.ui is a good id for ToDesk, it is more like a internal implemention detail choosen by todesk.com and may be changed in the future if the implemention changes. ToDesk is the product name for this app from todesk.com, I think com.todesk.ToDesk is better and more stable than com.todesk.toc.ui.